nixos/modules/tasks/filesystems.nix: Skip bindfs FS check

0xABAB opened the pull request:
###### Motivation for this change
A bindfs is a trivial FS. 

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Commit:
Skip bindfs FS check

Profpatsch commented:
Could you add a description on what this PR fixes to the commit message, please?

Profpatsch reviewed (changes requested):
Could you add a description on what this PR fixes to the commit message, please?

Commit:
Skip filesystem check for bindfs

Bindfs (FUSE) provides a pseudo-filesystem and as such does not benefit from a file system check.

Profpatsch commented:
Ah, you have two commits now. It should be possible to combine them with

```
git reset HEAD~1
git add  nixos/modules/tasks/filesystems.nix
git commit --amend
```

though it will drop your extended message on the first command.

Otherwise use `git rebase --interactive`.
